imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -36,7 +36,7 @@ class RibaMetaMod(Driver):
     has_metaswap: bool  # configured with or without metaswap
 
     max_iter: NDArray[Any]  # max. nr outer iterations in MODFLOW kernel
-    delt: float  # time step from MODFLOW 6 (leading)
+    delt: float = 1.0  # time step from MODFLOW 6 (leading), default 1.0
```

Comment:
this seems tricky. Normally MF updates this after the first init-timestep. Why this default than?